docs/getting_started.md

```diff
@@ -1,64 +1,53 @@
-# Get started
+# Getting Started with Streamlit
 
-The easiest way to learn how to use Streamlit is to try things out yourself. As
-you read through this guide, test each method. As long as your app is running,
-every time you add a new element to your script and save, Streamlit's UI will
-ask if you'd like to rerun the app and view the changes. This allows you to
-work in a fast interactive loop: you write some code, save it, review the
-output, write some more, and so on, until you're happy with the results. The
-goal is to use Streamlit to create an interactive app for your data or model
-and along the way to use Streamlit to review, debug, perfect, and share your
-code.
+The easiest way to learn Streamlit is to try things out yourself! As
+you work through this getting started guide, be sure to run each line of code.
+Each time you make a change to your script and save, the Streamlit UI will
+ask if you'd like to rerun the app and view the changes. This workflow allows for
+working in a fast, interactive loop: write some code, save it, review the
+output, write some more, and so on until you're happy with the results.
 
-Use the links below to jump to a specific section:
+## Part 1: Installing Streamlit in a virtual environment
```

Comment:
I don't think we should immediately throw people into the wild world of virtual environments here. This is a confusing topic for so many people, no need to make their Streamlit onboarding this much harder.

How about we tell them something like:
* To install Streamlit, just:
   ```
   pip install streamlit
   ```
   then try it out
   ```
   streamlit hello
   ```
* If you want to do virtual environments (recommended), do this instead: (...)

I understand the problem here is the Windows installation, which is made simpler once you have Conda. But I'm hoping we can find a middle-ground solution that doesn't just scare people away.

Reply:
One of the core points of this PR is actually disagreeing with that, that the virtual environment instructions should be separate. Having the reference to virtual environments hidden at https://docs.streamlit.io/en/latest/troubleshooting/clean-install.html makes it so people never see that it is the recommended practice.

I'm actually more in favor of removing the idea that we have officially supported installation methods at all.

Reply:
https://github.com/streamlit/streamlit/issues/1262#issuecomment-632741576

Reply:
Hey, just saw you updated this. Sorry for the delay! Please ping me next time you need a code review and it's been more than 24h :smiley: 

> Having the reference to virtual environments hidden at https://docs.streamlit.io/en/latest/troubleshooting/clean-install.html makes it so people never see that it is the recommended practice.

This makes total sense. I'm just trying to find a solution where we can both make that more visible and keep our installation instructions dead-easy.

Making Streamlit's installation and "time to first app" as simple/quick as possible is something we spent a long time working on last year, because simplicity and ease of use are actually some of our core value props. That's how we landed on "pip install streamlit + streamlit hello", for example.

LMK if you want to set up a quick chat to find ways to unblock this!

> I'm actually more in favor of removing the idea that we have officially supported installation methods at all.

Amen!

Some background on why we did that in the first place: that section was created to solve the problem of there being too many ways to install Python in Windows, and basically every method is super complex and full of pitfalls. We saw this first hand as lots of Windows users started pouring into the forums asking things like "I installed Streamlit but when I run `streamlit run foo.py` I'm told `streamlit` doesn't exist." The Conda approach is the most fool-proof solution we found at the time, so we wanted to call it out. 

That said, I'm happy to try something new to solve this problem. E.g. add more Windows-specific installation questions to FAQ?